types/jsonld/index.d.ts

```diff
@@ -132,6 +132,10 @@ export function normalize(input: JsonLdDocument, options: Options.Normalize, cal
 export function normalize(input: JsonLdDocument, callback: Callback<string>): void;
 export function normalize(input: JsonLdDocument, options?: Options.Normalize): Promise<string>;
 
+export function canonize(input: JsonLdDocument, options: Options.Normalize, callback: Callback<string>): void;
+export function canonize(input: JsonLdDocument, callback: Callback<string>): void;
+export function canonize(input: JsonLdDocument, options?: Options.Normalize): Promise<string>;
+
```

Comment:
```suggestion
export const canonize: typeof normalize;

```
IMO it would be better expressing `canonize` being just an alias to `normalize`.
The outcome would be same, no change required to tests:
```ts
(input: JsonLdDocument, options: Options.Normalize, callback: Callback<string>): void;
(input: JsonLdDocument, callback: Callback<string>): void;
(input: JsonLdDocument, options?: Options.Normalize | undefined): Promise<string>;
```

Reply:
great suggestion, thanks.